tests/feature/console/suseconnect.pm

```diff
@@ -0,0 +1,36 @@
+# Feature tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Case #1480023 : [316585] Drop suseRegister
+
+use base "consoletest";
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    select_console 'root-console';
+
+    #Check SUSEConnect is installed
+    script_run("rpm -q SUSEConnect | tee /dev/$serialdev");
+    wait_serial("SUSEConnect") || die "rpm -q SUSEConnect failed";
```

Comment:
Thanks @coolo , the API to verify is more efficient. May I simply rewrite it as following ? 

    #Check SUSEConnect is installed
    assert_script_run('rpm -q SUSEConnect');
    save_screenshot;

    #Check suseRegister is not installed
    assert_script_run('! rpm -q suseRegister');
    save_screenshot;

I've run it on openQA to retest again and passed. Although I am not sure is it a good way to add "!" to check the reverse. :-/ 
Thanks for reviewing.